cli/deal/showdeals.go

```diff
@@ -0,0 +1,86 @@
+package deal
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/fatih/color"
+	lapi "github.com/filecoin-project/lotus/api"
+)
+
+var mockDealInfos []lapi.DealInfo
+
+func InspectDealCmd(ctx context.Context, api lapi.FullNode, proposalCid string, dealId int) error {
+	ctx, cancel := context.WithCancel(ctx)
+	defer cancel()
+
+	deals, err := api.ClientListDeals(ctx)
+	if err != nil {
+		return err
+	}
+
+	var di lapi.DealInfo
```

Comment:
This could probably be a pointer, and then we can remove `found` and do a nil check below to know if the deal was found or not. Although be careful with local variables in loops.

Reply:
Done.